scripts/parts.py

```diff
@@ -24,18 +24,24 @@
     parser.add_argument('--port', help="Specify port [all]", default='all')
     parser.add_argument('-e', '--exact-match', help="Force exact matches on part numbers, not beginning N char. [False]",
                         dest='exact_match', action='store_true')
-    parser.add_argument('--notes', help="(For action=part_info):  Displays part notes with posting dates and not other info",
+    parser.add_argument('--notes', help="<For action=part_info>:  Displays part notes with posting dates and not other info",
                         action='store_true')
+    parser.add_argument('--sort_notes_by', help="<For action=part_info --notes> Can sort the notes listing by part \
+                        ('part') or posting time ('post') [part]", default='part')
     cm_utils.add_verbosity_args(parser)
     cm_utils.add_date_time_args(parser)
+    parser.add_argument('--notes_start_date', help="<For part_info notes> start_date for notes [<]", default='<')
+    parser.add_argument('--notes_start_time', help="<For part_info notes> start_time for notes", default=0.0)
```

Comment:
For a script, I'd think it'd be a little easier to be able to enter a date and time together rather than having to specify two separate keywords. Astropy Time can certainly handle just being passed a date string with no time info -- it just uses midnight as the time if it's not specified, which I think is the same as your default here.

Reply:
That would make it inconsistent with every other cm script.  If you think we should change this, can you make an issue then I'll go through and change all?

Reply:
I don't feel super strongly about this and I agree consistency is more important. Happy to review a PR if you decide you want to make the change across all of them.